install_config/redeploying_certificates.adoc

```diff
@@ -0,0 +1,73 @@
+[[install-config-redeploying-certificates]]
+= Redeploying Certificates
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+This topic reviews how to back up and redeploy cluster certificates using
+`ansible-playbook`. This method fixes common certificate errors. Possible use
```

Comment:
Would it be more accurate to say something like "ansible-playbook command" or "Ansible playbook"?